volume/local/local_unix.go

```diff
@@ -71,6 +70,30 @@ func (v *localVolume) mount() error {
 	if v.opts.MountDevice == "" {
 		return fmt.Errorf("missing device in volume options")
 	}
+	r1, _ := regexp.Compile(`\baddr=*`)
+	r2, _ := regexp.Compile(`\bnfs\b`)
+	if r1.MatchString(v.opts.MountOpts) && r2.MatchString(v.opts.MountType) {
+		if addrValue := getAddress(v.opts.MountOpts); addrValue != "" {
+			ipAddr, err := net.ResolveIPAddr("ip", addrValue)
+			if err == nil {
+				v.opts.MountOpts = strings.Replace(v.opts.MountOpts, addrValue, ipAddr.String(), 1)
```

Comment:
Should we replace the value of `v.opts.MountOps` here, or should we use a copy, and leave the original options as-is? @cpuguy83 ?